clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/cache/Keys.groovy

```diff
@@ -53,6 +53,10 @@ class Keys {
       return null
     }
 
+    if (parts[0] != GoogleCloudProvider.GCE) {
```

Comment:
Bug in GCE caching that was causing it to crash when other providers are enabled.